content/en/agent/kubernetes/helm.md

```diff
@@ -138,9 +138,51 @@ datadog:
  logsConfigContainerCollectAll: true
 ```
 
+### Enable APM and Distributed Tracing
+
+Update your [datadog-values.yaml][7] file with the following APM configuration:
+
+```
+datadog:
+  (...)
+  apmEnabled: true
+
+(...)
+
+daemonset:
+  (...)
+  useHostPort: true
+```
+
+Use the downward API to pull the host IP; the application container needs an environment variable that points to `status.hostIP`. You can name it `DD_AGENT_HOST`. Update the `env` section of your `datadog-values.yaml` file with the following:
```

Comment:
For the helm chart configuration, this is actually being done on the user’s application side. So we might want to say something similar to what you did for the Daemonset section. Maybe something like `Update the env section of your application's manifest, or values.yaml file with the following:`